codacy/values-production.yaml

```diff
@@ -7,6 +15,10 @@ global:
     # protocol: "https"
     # disableSSL: "false"
     # isPrivateMode: "true"
+    # app:
+    #   appId: "1234"
+    #   TODO: Add link to documentation
```

Comment:
Remove this and create a task to track that its done when the docs are available